first draft of GTA 2026 event page (#6611)
galaxyproject/training-material: events/2025-05-18-galaxy-academy.md

---
layout: event

title: Galaxy Training Academy 2026
description: |

>   The Galaxy Training Academy is a self-paced online training event for beginners and advanced learners who want to improve their Galaxy data analysis skills.

Reviewer: ```suggestion
  The Galaxy Training Academy is a self-paced online training event for beginners and advanced learners who want to improve their skills in data analysis in Galaxy and/or in popular fields in bioinformatics.
```

#        - subinamehta
#        - teresa-m
#        - timothygriffin
#        - tcollins2011
#        - nomadscientist
#        - wm75
#    funding:
#        - eurosciencegateway
#        - biont
#        - nfdi4plants
#        - deNBI
#        - elixir-europe
#        - mwk
#        - abromics
#        - ifb
#        - FAIR2Adapt

location:
  name: online

infrastructure:
  tiaas: true
  servers:
    - server: https://usegalaxy.eu
      name: Galaxy EU
      # tiaas_link:
    - server: https://usegalaxy.org
      name: Galaxy Main
    - server: https://usegalaxy.org.au/
      name: Galaxy AU
    - server: https://usegalaxy.fr
      name: Galaxy FR
      tiaas_link: https://usegalaxy.fr/join-training/gta2025
  support:
    platform: Slack

---
# Welcome to the Galaxy Training Academy 2026
Do you want to learn how to use Galaxy, an open source data analysis platform? Then you are at the right place. We offer here a **5-day Global Online and Asynchronous learning event**.

**Content of the event**

We provide you with training materials which you can study at your own pace and on your own time throughout the week. A program will be posted here closer to the event start.

> On the first day, you can make yourself familiar with the Galaxy platform. In the next days, you can learn about Proteomics, Assembly, Transcriptomics, Single Cell, Microbiome or Machine Learning data analysis in Galaxy. On the last day, the Friday we additionally offer a ***FAIR training*** program to you.

Reviewer: ```suggestion
On the first day, you can make yourself familiar with the Galaxy platform. In the next days, you can learn about Proteomics, Assembly, MAchine learnign,  and other tracks that will be defined as we get closer to the event. 
```

**Course format**

The event is asynchronous which means you can set your own pace on your learning journey using our provided self-learning materials. You will start the event by yourself at your preferred time.
Don't worry, asynchronous does not mean that you are alone! If you ever need help, you can contact one of our many trainers worldwide via **Slack chat**.

**How to participate**

First, you will need to register. **We will open registrations soon.**
You only need a browser and an account on one of the Galaxy instances registered for this event.

**How to get help**

You will not be alone! If you ever need help, you can contact one of our many trainers worldwide via **Slack chat**. Next to the program, you will find Slack channels you can join to exchange with the trainers and other participants during the event. Here you will also find help if you have questions or run into an issue during the training. We try to cover all time zones with helpers for each topic, but please be patient if you do not get an immediate response.

**Certificates**

> You will be able to obtain a certificate by the end of the event. More information is coming soon.

Reviewer: ```suggestion
You will be able to obtain a certificate by the end of the event. To ensure you have all the time you need to learn at your own pace, you will be able to claim a certificate for a few weeks after the end of the official event. More information is coming soon.
```
